src/shared/utils/Slot.test.tsx

```
@@ -0,0 +1,99 @@
+  asChild,
+  children,
+  ...restProps
+}: ButtonHTMLAttributes<HTMLButtonElement> & { asChild?: boolean }) {
+  const Element = asChild ? Slot : 'button'
+
+  return (
+    <Element {...restProps}>
+      <Slottable>{children}</Slottable>
+    </Element>
+  )
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)